modules/examples/src/main/java/com/ibm/cloud/platform_services/enterprise_management/v1/EnterpriseManagementExamples.java

```diff
@@ -0,0 +1,357 @@
+    }
+
+    try {
+      // begin-create_account
+      String parent = "crn:v1:bluemix:public:enterprise::a/%s::account-group:%s";
+      CreateAccountOptions createAccountOptions = new CreateAccountOptions.Builder()
+          .parent(String.format(parent, enterpriseAccountId, firstAccountGroupId))
+          .name(accountName)
+          .ownerIamId(enterpriseAccountIamId)
+          .build();
+
+      Response<CreateAccountResponse> response = service
+          .createAccount(createAccountOptions)
+          .execute();
+      CreateAccountResponse createAccountResponse = response.getResult();
+
+      System.out.println(createAccountResponse);
+      // end-create_account
+      accountId = createAccountResponse.getAccountId();
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s%nError details: %s", e.getStatusCode(),
+          e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-list_accounts
+      ListAccountsOptions listAccountsOptions = new ListAccountsOptions.Builder()
+          .enterpriseId(enterpriseId)
+          .build();
+
+      Response<ListAccountsResponse> response = service
+          .listAccounts(listAccountsOptions)
+          .execute();
+      ListAccountsResponse listAccountsResponse = response.getResult();
+
+      System.out.println(listAccountsResponse);
+      // end-list_accounts
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s%nError details: %s", e.getStatusCode(),
+          e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-get_account
+      GetAccountOptions getAccountOptions = new GetAccountOptions.Builder()
+          .accountId(accountId)
+          .build();
+
+      Response<Account> response = service
+          .getAccount(getAccountOptions)
+          .execute();
+      Account account = response.getResult();
+
+      System.out.println(account);
+      // end-get_account
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s%nError details: %s", e.getStatusCode(),
+          e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-update_account
+      String newParent = "crn:v1:bluemix:public:enterprise::a/%s::account-group:%s";
+      UpdateAccountOptions updateAccountOptions = new UpdateAccountOptions.Builder()
+          .accountId(accountId)
+          .parent(String.format(newParent, enterpriseAccountId, secondAccountGroupId))
+          .build();
+
+      service
+          .updateAccount(updateAccountOptions)
+          .execute();
+      // end-update_account
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s%nError details: %s", e.getStatusCode(),
+          e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-create_enterprise
```

Comment:
```suggestion
      String accountId = "example_account_id";
      String contactIamId = "example_iam_id";
      // begin-create_enterprise
```

Reply:
`accountId` is already defined, so I used `sourceAccountId` or something else in which we agree with Norbert.